tor/core/inbox.py

```diff
@@ -75,7 +75,6 @@ def process_reply(reply, config):
         if any([regex.search(reply.body) for regex in MOD_SUPPORT_PHRASES]):
             process_mod_intervention(reply, config)
             reply.mark_read()
```

Comment:
I would also remove this line so that there wouldn’t be issues if we try to mark it as read twice.

I imagine we would want the full messages caught in the unhandled message handler that make it through here.